modules/apps/object/object-api/src/main/java/com/liferay/object/constants/ObjectPortletKeys.java

```diff
@@ -22,4 +22,8 @@ public class ObjectPortletKeys {
 		"com_liferay_object_web_internal_object_entries_portlet_" +
 			"ObjectEntriesPortlet";
 
+	public static final String OBJECT_FOLDERS =
+		"com_liferay_object_web_internal_object_folders_portlet_" +
+			"ObjectFoldersPortlet";
```

Comment:
Here you are referring a portlet that doesn't exists, are you creating this portlet in another PR?

Reply:
Can you check if we can use the `ObjectDefinitionsPortlet?`

Reply:
Thanks, we absolutely can.